integration-test/common.ps1

```diff
@@ -175,7 +175,7 @@ BeforeAll {
     function DotnetNew([string] $type, [string] $name, [string] $framework)
     {
         Remove-Item -Path $name -Recurse -Force -ErrorAction SilentlyContinue
-        dotnet new $type --output $name --framework $framework | ForEach-Object { Write-Host $_ }
+        dotnet new $type --output $name --framework $framework --no-restore | ForEach-Object { Write-Host $_ }
```

Comment:
thought: This may break existing usages of `DotnetNew` that may depend on the _implicit restore_.

Could/should we make `--no-restore` and optional parameter,
where the default is executing the _implicit restore_
and when explicitly set (to `true`), then we pass `--no-restore`.

Reply:
I didn't realize it was template-specific. `dotnet new console` has the `--no-restore` option but `dotnet new maui` doesn't :expressionless:

Reply:
oh really?! ... I also thought this was an option on `dotnet new`, and not an individual option per _template_ 🤯